so, can someone help me figure out that flake8 error?

```
flake8 --ignore=E127,E203,W503,F811 --show-source ./iocage/lib/ioc_stop.py 
./iocage/lib/ioc_stop.py:1:7: F821 undefined name 'uuid'
# Copyright (c) 2014-2017, iocage
      ^
```

--- a/fastentrypoints.py
+++ b/fastentrypoints.py
@@ -99,9 +99,10 @@ def main():
         with open(setup_path, 'a+') as setup:
             setup.seek(0)
             setup_content = setup.read()
-            if not 'import fastentrypoints' in setup_content:
+            if 'import fastentrypoints' not in setup_content:
                 setup.seek(0)
                 setup.truncate()
                 setup.write('import fastentrypoints\n' + setup_content)
+
 
 print(__name__)

--- a/iocage/lib/ioc_check.py
+++ b/iocage/lib/ioc_check.py
@@ -101,24 +101,27 @@ class IOCCheck(object):
         Checks if /dev/fd is mounted, and if not, give the user a
         warning.
         """
-        if not os.path.ismount("/dev/fd"):
-            messages = collections.OrderedDict([
-                ("1-NOTICE", "*" * 80),
-                ("2-WARNING", "fdescfs(5) is not mounted, performance "
-                              " may suffer. Please run:"),
-                ("3-INFO", "mount -t fdescfs fdesc /dev/fd"),
-                ("4-WARNING", "You can also permanently mount it in "
-                              "/etc/fstab with the following entry:"),
-                ("5-INFO", "fdesc /dev/fd  fdescfs  rw  0  0"),
-                ("6-NOTICE", f"{'*' * 80}\n")
-            ])
+        if os.path.ismount("/dev/fd"):
+            # all good!
+            return
 
-            for level, msg in messages.items():
-                level = level.partition("-")[2]
+        messages = collections.OrderedDict([
+            ("1-NOTICE", "*" * 80),
+            ("2-WARNING", "fdescfs(5) is not mounted, performance "
+                            " may suffer. Please run:"),
+            ("3-INFO", "mount -t fdescfs fdesc /dev/fd"),
+            ("4-WARNING", "You can also permanently mount it in "
+                            "/etc/fstab with the following entry:"),
+            ("5-INFO", "fdesc /dev/fd  fdescfs  rw  0  0"),
+            ("6-NOTICE", f"{'*' * 80}\n")
+        ])
 
-                iocage.lib.ioc_common.logit({
-                    "level"  : level,
-                    "message": msg
-                },
-                    _callback=self.callback,
-                    silent=self.silent)
+        for level, msg in messages.items():
+            level = level.partition("-")[2]
+
+            iocage.lib.ioc_common.logit({
+                "level"  : level,
+                "message": msg
+            },
+                _callback=self.callback,
+                silent=self.silent)

--- a/iocage/lib/ioc_common.py
+++ b/iocage/lib/ioc_common.py
@@ -439,43 +439,44 @@ def git_pull(repo, remote_name="origin", branch="master"):
     # https://raw.githubusercontent.com/MichaelBoselowitz/pygit2-examples/master/examples.py
     # @formatter:on
     for remote in repo.remotes:
-        if remote.name == remote_name:
-            remote.fetch()
-            remote_master_id = repo.lookup_reference(
-                f"refs/remotes/origin/{branch}").target
-            merge_result, _ = repo.merge_analysis(remote_master_id)
-            # Up to date, do nothing
-            if merge_result & pygit2.GIT_MERGE_ANALYSIS_UP_TO_DATE:
-                return
-            # We can just fastforward
-            elif merge_result & pygit2.GIT_MERGE_ANALYSIS_FASTFORWARD:
-                repo.checkout_tree(repo.get(remote_master_id))
-                try:
-                    master_ref = repo.lookup_reference(f'refs/heads/{branch}')
-                    master_ref.set_target(remote_master_id)
-                except KeyError:
-                    repo.create_branch(branch, repo.get(remote_master_id))
-                repo.head.set_target(remote_master_id)
-            elif merge_result & pygit2.GIT_MERGE_ANALYSIS_NORMAL:
-                repo.merge(remote_master_id)
+        if remote.name != remote_name:
+            continue
 
-                if repo.index.conflicts is not None:
-                    for conflict in repo.index.conflicts:
-                        logit({
-                            "level"  : "EXCEPTION",
-                            "message": "Conflicts found in:"
-                                       f" {conflict[0].path}"
-                        })
+        remote.fetch()
+        remote_master_id = repo.lookup_reference(
+            f"refs/remotes/origin/{branch}").target
+        merge_result, _ = repo.merge_analysis(remote_master_id)
+        # Up to date, do nothing
+        if merge_result & pygit2.GIT_MERGE_ANALYSIS_UP_TO_DATE:
+            return
+        # We can just fastforward
+        elif merge_result & pygit2.GIT_MERGE_ANALYSIS_FASTFORWARD:
+            repo.checkout_tree(repo.get(remote_master_id))
+            try:
+                master_ref = repo.lookup_reference(f'refs/heads/{branch}')
+                master_ref.set_target(remote_master_id)
+            except KeyError:
+                repo.create_branch(branch, repo.get(remote_master_id))
+            repo.head.set_target(remote_master_id)
+        elif merge_result & pygit2.GIT_MERGE_ANALYSIS_NORMAL:
+            repo.merge(remote_master_id)
 
-                user = repo.default_signature
-                tree = repo.index.write_tree()
-                repo.create_commit('HEAD', user, user, "merged by iocage",
-                                   tree, [repo.head.target, remote_master_id])
-                # We need to do this or git CLI will think we are still
-                # merging.
-                repo.state_cleanup()
-            else:
-                raise AssertionError("Unknown merge analysis result")
+            if repo.index.conflicts is not None:
+                for conflict in repo.index.conflicts:
+                    logit({
+                        "level"  : "EXCEPTION",
+                        "message": "Conflicts found in: {conflict[0].path}"
+                    })
+
+            user = repo.default_signature
+            tree = repo.index.write_tree()
+            repo.create_commit('HEAD', user, user, "merged by iocage",
+                                tree, [repo.head.target, remote_master_id])
+            # We need to do this or git CLI will think we are still
+            # merging.
+            repo.state_cleanup()
+        else:
+            raise AssertionError("Unknown merge analysis result")
 
 
 def set_rcconf(jail_path, key, value):

--- a/iocage/lib/ioc_destroy.py
+++ b/iocage/lib/ioc_destroy.py
@@ -48,24 +48,23 @@ class IOCDestroy(object):
     @staticmethod
     def __stop_jails__(datasets, path=None, root=False):
         for dataset in datasets.dependents:
-            if "jails" in dataset.name:
-                # This is just to setup a replacement.
-                path = path.replace("templates", "jails")
-                uuid = dataset.name.partition(f"{path}/")[2].rsplit("/", 1)[0]
-                # We want the real path now.
-                if dataset.type == libzfs.DatasetType.FILESYSTEM:
-                    _path = dataset.properties["mountpoint"].value.replace(
-                        "/root", "")
-                    # It gives us a string that says "none", not terribly
-                    # useful, fixing that.
-                    _path = _path if _path != "none" else None
+            if "jails" not in dataset.name:
+                continue
+            if dataset.type != libzfs.DatasetType.FILESYSTEM:
+                continue
 
-                    if dataset.name.endswith(uuid) or root:
-                        if _path is not None:
-                            conf = iocage.lib.ioc_json.IOCJson(
-                                _path).json_load()
-                            iocage.lib.ioc_stop.IOCStop(uuid, _path, conf,
-                                                        silent=True)
+            # This is just to setup a replacement.
+            path = path.replace("templates", "jails")
+            uuid = dataset.name.partition(f"{path}/")[2].rsplit("/", 1)[0]
+            # We want the real path now.
+            _path = dataset.properties["mountpoint"].value.replace("/root", "")
+            # It gives us a string that says "none", not terribly
+            # useful, fixing that.
+            _path = _path if _path != "none" else None
+
+            if (dataset.name.endswith(uuid) or root) and _path is not None:
+                conf = iocage.lib.ioc_json.IOCJson(_path).json_load()
+                iocage.lib.ioc_stop.IOCStop(uuid, _path, conf, silent=True)
 
     def __destroy_leftovers__(self, dataset, clean=False):
         """Removes parent datasets and logs."""

